gatling-core/src/main/scala/io/gatling/core/structure/Errors.scala

```diff
@@ -17,14 +17,14 @@ package io.gatling.core.structure
 
 import java.util.UUID
 
+import io.gatling.commons.validation.Success
 import io.gatling.core.action.builder.{ ExitHereIfFailedBuilder, TryMaxBuilder }
+import io.gatling.core.session.Expression
 
 trait Errors[B] extends Execs[B] {
 
-  def exitBlockOnFail(chain: ChainBuilder): B = tryMax(1)(chain)
-  def tryMax(times: Int, counter: String = UUID.randomUUID.toString)(chain: ChainBuilder): B = {
-
-    require(times >= 1, "Can't set up a max try <= 1")
+  def exitBlockOnFail(chain: ChainBuilder): B = tryMax(_ => Success(1))(chain)
```

Comment:
Use `expressionSuccess` from the `session` package object so it doesn't instantiate a `Success` object every time the function is applied.